wiki/Installation/macOS/en.md

```diff
@@ -1,20 +1,43 @@
+The macOS version of osu! is subject to minor display bugs and less performant gameplay. The game may not function perfectly at first and takes some fine-tuning to get used to. To install it:
 
-1. Go to the unofficial [download page](https://osu.ppy.sh/community/forums/topics/682197), download `osu!.app.zip` and unzip it.
-2. You will see the `osu!.app` folder, which will look like an application. Right click it, and select `Open`.
-3. The installer will automatically run. The installation location cannot be changed and will be installed directly to `osu!.app`. Moving `osu!.app` will move all of the files that come with it. You can also right click it and select `Show Package Contents` to view the files within the folder.
-4. When the installer is completed, osu! will start up right after. osu!direct will be used to download some starter beatmaps for you.
-   - If osu! is blocked from opening due to having an unidentified developer, open `System Preferences > Security and Privacy` and click `Open Anyway`.
-5. osu! will prompt you to sign in or register.
+1. Go to the unofficial Wineskin [download page](https://osu.ppy.sh/community/forums/topics/1106057), download the latest Wineskin and unzip it.
+2. Download the unofficial [osu!macOS Agent](https://osu.ppy.sh/community/forums/topics/1036678) tool and use it to repair the `osu!.app` file. Alternatively:
+   1. Move `osu!.app` to the desktop.
+   2. Open Terminal, copy this command: `xattr -c 'Desktop/osu!.app'`, and press enter.
+3. Double click on `osu!.app` to open it.
+4. The installer will automatically run. The installation location cannot be changed and will reside directly inside `osu!.app`. Moving `osu!.app` will move all of the files that come with it including any [beatmaps](/wiki/Beatmaps) and [skins](/wiki/Skin). You can also right click it and select `Show Package Contents` to view the files within the folder.
+5. When the installation is complete, osu! will start up right after. osu!direct will be used to download some starter beatmaps for you.
+6. osu! will prompt you to sign in or register.
    - If you have an account already, sign in.
    - If you need to register, see [Registration](/wiki/Registration), then sign in.
+
+## Troubleshooting
+
+If you have difficulties opening osu! then the following steps may help you:
+
+1. Check the report generated by [osu!macOS Agent](https://osu.ppy.sh/community/forums/topics/1036678) for any errors.
+   - If you are on an older version of macOS Catalina (before or equal to 10.15.4) then you must either update macOS (recommended) or disable [System Integrity Protection](https://developer.apple.com/documentation/security/disabling_and_enabling_system_integrity_protection).
+2. Use the extractor "Archive Utility" builtin to macOS. Other extractors may not unzip the Wineskin correctly.
+3. If you have an antivirus installed ensure that it is disabled when you unzip osu! or else it may quarantine files required by Wine.
```

Comment:
```suggestion
3. If you have an antivirus installed, ensure that it is disabled when you unzip osu! or else it may quarantine files required by Wine.
```